APISpec.md

```diff
@@ -156,6 +156,30 @@ Adopts a creature. User's affinity level with the specified creature must be 100
     "success": "boolean"
 }
 ```
+### 2.4 Breed Creatures `/users/{user_id}/creatures/breed` (POST)
+
+Breeds 2 creatures together. Creatures must be adopted by the user.

```

Comment:
I would add something here to clarify that the "name" part of the request body is for the child creature's name